src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/TrafficSeo/Meta/UrlSchemaType.php

```diff
@@ -62,4 +126,34 @@ public function configureOptions(OptionsResolver $resolver)
             'label' => false,
         ]);
     }
+
+    /**
+     * @param string $idRoute
+     *
+     * @return string
+     *
+     * @throws \PrestaShopException
+     */
+    private function getKeywords($idRoute)
+    {
+        $keyWords = $this->defaultRouteProvider->getKeywords();
+        $formattedKeyWords = [];
+        if ($keyWords[$idRoute]) {
+            foreach ($keyWords[$idRoute] as $key => $keyWord) {
+                $value = $key;
+                if (isset($keyWord['param'])) {
+                    $value .= '*';
+                }
+                $formattedKeyWords[] = $value;
+            }
+        }
+
+        return sprintf(
+            $this->trans(
+                'Keywords: %s',
```

Comment:
@JevgenijVisockij, it is likely that it should be replaced by _Keywords: %keywords%_ (still localized in _Admin.Shopparameters.Feature_), this string does not exist since the 1.7.3 catalog, cf. https://github.com/PrestaShop/PrestaShop/pull/21472#issuecomment-710103625